Sources/TensorFlow/Operators/Basic.swift

```diff
@@ -575,6 +575,15 @@ public extension Tensor {
     }
 }
 
+public extension Tensor where Scalar: Numeric {
+    /// Returns a `Tensor` by clipping tensor values to a specified min and max.
+    @inlinable
+    @differentiable(wrt: self where Scalar: TensorFlowFloatingPoint)
```

Comment:
This will not compile because `Raw` functions are not differentiable and you haven't defined a VJP for this yet.  Please comment out the `@differentiable` attribute and add a FIXME.
```suggestion
    // FIXME: Define a derivative function.
    // @differentiable(wrt: self where Scalar: TensorFlowFloatingPoint)
```

Reply:
@rxwei I'll do this, but how does one define a vjp for clipped function?

Reply:
Please file an issue for this and move the discussion there. Thanks!